data.sql

```diff
@@ -11,3 +11,27 @@ VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
 
 INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
 VALUES ('Devimon', '2017-05-12', 5, true, 11);
+
+/* Upload the data */
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', '2020-02-08', 0, false, -11.0);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', '2021-11-15', 2, true, -5.7);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Squirtle', '1993-04-02', 3, false, -12.13);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Angemon', '2005-06-12', 1, true, -45.0);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Boarmon', '2005-06-07', 7, true, 20.4);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Blossom', '1998-10-13', 3, true, 17.0);
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Ditto', '2022-05-14', 4, true, 22);
+
+SELECT * FROM animals;
```

Comment:
- [ OPTIONAL ] 👋 Hey there! Let's make sure that we've got all our queries neatly organized in the "queries.sql" file 📁. This will help keep our project well-structured and make it easier for everyone to locate and manage the queries efficiently. Thank you for your attention to this important detail! 🙌😊

Reply:
Done, thanks for you review 👍😃